Common/src/CConfig.cpp

```diff
@@ -1150,6 +1150,8 @@ void CConfig::SetConfig_Options() {
   /*!\par CONFIG_CATEGORY: FluidModel \ingroup Config*/
   /*!\brief FLUID_MODEL \n DESCRIPTION: Fluid model \n OPTIONS: See \link FluidModel_Map \endlink \n DEFAULT: STANDARD_AIR \ingroup Config*/
   addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, STANDARD_AIR);
+  /*!\brief FLUID_NAME \n DESCRIPTION: Fluid name \n OPTIONS: see coolprop homepage \n DEFAULT: nitrogen \ingroup Config*/
+  addStringOption("FLUID_NAME", FluidName, string("nitrogen"));
```

Comment:
Sure, I can try.

However, I do not find GAS_MODEL option in CConfig.hpp and option_structure.hpp.

could you please let me know where I can find GAS_MODEL option so that I can use it.

Reply:
It's in CConfig.cpp

Reply:
Hi, may I re-use the GAS_MODEL option later?

I am working on introducing viscosity model and conductivity model from CoolProp and it is almost done.

So my plan is: after this PR is approved, then I open a new PR about viscosity and conductivity model. Once the new PR  is approved, then I work on the re-use of  GAS_MODEL option.

Do you have any suggestion?

Thank you.

Reply:
No that's fine, there are some checks on GAS_MODEL specific to mutation++ so it may be simpler to just have two options.